routes/userRouteNotes.js

```diff
@@ -0,0 +1,57 @@
+router.get('/widgets/:widgetId', ( request, response, next ) => {
+  const { widgetId } = request.params
+
+  User.find( {'widgets._id': widgetId},function(error, widgets){
```

Comment:
Use ES6, which will allow a single line, implicit return.  Formatting is also off.